src/Whoops/Run.php

```diff
@@ -293,6 +293,24 @@ public function handleException(Exception $exception)
     }
 
     /**
+     * This is a private implementation detail, you are not supposed to call this.
+     */
+    final public function handleExceptionFromPHP($exception)
```

Comment:
I thought about that, but it is so clunky. And for what — to avoid people shooting themselves in the face ignoring the name of the function, its absence from the documentation and the comment forbidding its use? Nah.